src/domain/eventSearch/utils.tsx

```diff
@@ -263,30 +311,41 @@ export const getEventSearchVariables = ({
     keywordAnd.push('yso:p4354');
   }
 
-  const keywordsParamName = MAPPED_KEYWORD_TERMS[searchSource];
-  const mappedCategories: string[] = categories
-    .map((category) => MAPPED_CATEGORIES[searchSource][category])
-    .filter((e) => e);
-  const mappedHobbyTypes: string[] =
-    hobbyTypes
-      ?.map((hobbyType) => MAPPED_COURSE_HOBBY_TYPES[hobbyType])
-      .filter((e) => e) || [];
+  const categoriesParamName = MAPPED_KEYWORD_TERMS[searchSource];
+  let mappedCategories: string[] = [];
+  categories.map(
+    (category) =>
+      (mappedCategories = mappedCategories
+        .concat(MAPPED_CATEGORIES[searchSource][category]?.split(','))
+        .filter((e) => e))
```

Comment:
Could we just assume there are no falsy values?

Reply:
I mean do necessarily need .filter here?

Reply:
oh looks like we do need it, but I think this will be enough also:

`prev.concat(categoryMap[category]?.split(',') ?? [])`

Reply:
yea, there will be falsy. and again ??. will refactor it